README.FreeBSD

```diff
@@ -11,152 +11,112 @@ In the end the cluster would be running bhyve on RBD disk that are stored in
 Ceph.
 
 The FreeBSD build will build most of the tools in Ceph.
-	
+
 Progress from last report:
 ==========================
-
-Most important change:
-  -	A port is submitted: net/ceph-devel.
+     
+Most important changes:
+ - Ceph has releassed the release candidate of v12.1.0 (aka Luminous),
+   the port is sitting in my tree waiting for Luminous to be actually 
+   released.
+ - ceph-fuse works, and allows mounting of cephfs filesystems. 
+   Speed is not impressive, but is does work.
+ - rbd-ggate available to create a Ceph rdb backed device. 
```

Comment:
@amitkumar50 
According to the Oxford dictionary is 'back' a verb, so backed would be correct grammar.
But I'll fix it.